client/components/theme/index.jsx

```diff
@@ -172,8 +176,12 @@ const Theme = React.createClass( {
 								context: 'singular noun, the currently active theme'
 							} ) }</span>
 						}
-						{ price && ! purchased &&
-							<span className="theme-badge__price">{ price }</span>
+						{
+							config.isEnabled( 'jetpack/pijp' )
+							? ! this.props.isJetpack && price && ! purchased &&
```

Comment:
@folletto Thanks! What about theme thumbnails (in the themes list), do we also display "PLAN" instead of the price there?

![image](https://user-images.githubusercontent.com/96308/27283676-a4468444-54f5-11e7-8526-c528290930e8.png)